app/page.tsx

```diff
@@ -25,8 +29,10 @@ export default function Home() {
         placeHolder={"Please Input Search Text"}
         disabled={false}
         readOnly={false}
+        icon={<FiSearch className="textField_icon" />}
         onChange={handleOnChangeInput}
```

Comment:
challenge: Imagine I want to pass more attributes to the field's input element, eg:

```
<TextField
  id={"book-search-input"}
  labelText={"Book Store"}
  // ...
  data-something="1"
/>
```

How could you adapt the Component to accept any prop without explicit declare it? ;)

Reply:
not clear on this one but we can meet next time to talk about this